code/src/java/pcgen/gui2/tabs/skill/SkillTreeViewModel.java

```diff
@@ -209,11 +209,7 @@ public void skillBonusChanged(CharacterLevelEvent e)
 		Object displayPath[];
 		if (path.length > 0 && StringUtils.isEmpty(String.valueOf(path[path.length - 1])))
 		{
-			displayPath = new Object[path.length - 1];
```

Comment:
Yup, this breaks the skill tree view TYPE/NAME. I have submitted RP #1910 to fix it.